cpp/test/IceSSL/configuration/SecureTransportTests.cpp

```diff
@@ -150,9 +155,35 @@ clientRejectsServerSettingTrustedRootCertificates(Test::TestHelper* helper, cons
 }
 
 void
-clientRejectsServerUsingDefaultTrustedRootCertificates(Test::TestHelper* helper, const string& certificatesPath)
+clientValidatesServerUsingSystemTrustedRootCertificates(Test::TestHelper*, const string&)
 {
-    cout << "client rejects server using default trusted root certificates... " << flush;
+    cout << "client validates server using system trusted root certificates... " << flush;
+    Ice::SSL::ClientAuthenticationOptions clientAuthenticationOptions{
+        .sslNewSessionCallback = [](SSLContextRef ssl, const string& host)
```

Comment:
Ditto. Does the already somewhat configured context ref so have SNI set?

Reply:
It doesn't